packages/ui-components/src/components/text-field/text-field.utils.ts

```diff
@@ -19,3 +20,15 @@ export function getInputMaskConfig(type: EInputFieldType): Inputmask.Options {
 export function isInputMaskCompatibleType(type: EInputFieldType): boolean {
 	return [EInputFieldType.Number, EInputFieldType.Text, EInputFieldType.DateTime].includes(type);
 }
+
+export function buildInputMask(input: HTMLInputElement, inputType: EInputFieldType, options: Inputmask.Options, maxLength?: number): IInputMaskInstanceRef {
+	// @ts-ignore the types library for Inputmask has the callback type definition for onBeforePaste incorrect, it should return string | false
```

Comment:
to avoid this ts-ignore can we map in `IInputMaskInstanceRef` the onBeforePaste to the correct type?